build.gradle

```diff
@@ -20,63 +20,94 @@ plugins {
     id "com.github.ben-manes.versions" version '0.38.0'
 }
 
-apply plugin: "com.vanniktech.maven.publish"
 
-repositories {
-    mavenCentral()
-    jcenter()
-}
 
-testSets {
-    integrationTest
-}
 
-sourceSets {
-    test {
-        resources {
-            srcDir 'src/testFixtures/resources'
+allprojects {
+    apply plugin: "com.vanniktech.maven.publish"
```

Comment:
Why this plugin can not be included in "plugins" block?

Reply:
It wasn't like that because this plugin is not in default gradle plugin repository. But I've added it and it should work just like the other plugins. Also `plugins {}` is not supported in `allprojects` or `subprojects` so I have to still use old `apply plugin`

Reply:
ok